source/_integrations/freebox.markdown

```diff
@@ -18,6 +18,7 @@ There is currently support for the following device types within Home Assistant:
 * [Sensor](#sensor) with traffic metrics
 * [Device tracker](#presence-detection) for connected devices
 * [Switch](#switch) to control WiFi
+* [Service](#service) to reboot the Freebox router
```

Comment:
Service is not an entity type, please remove it from this list. (This is not a TOC).

Reply:
removed 👍